pkg/storage/backend-storage/backend-storage.go

```diff
@@ -413,6 +417,29 @@ func (bs *BackendStorage) getStorageClass() (string, error) {
 	}
 }
 
+func (bs *BackendStorage) getBackendPVCSize(storageClassName string) (*resource.Quantity, error) {
+	// Default size is 10Mi if the annotation is not set
+	defaultQuantity := resource.MustParse(DefaultPVCSize)
+
+	obj, exists, err := bs.scStore.GetByKey(storageClassName)
+	if err != nil || !exists {
+		return &defaultQuantity, err
```

Comment:
I wonder if we should as some V3 logging that we choose default size since we didnt find the sc?

Reply:
ack